packages/commonwealth/client/styles/pages/manage_community/upgrade_roles_form.scss

```diff
@@ -13,6 +13,7 @@
     height: 200px;
     overflow-y: auto;
     padding: 0 8px;
+    width: 100%;
```

Comment:
same as above: this will effect existing components, lets make sure we use different css value for width based on the feature flag, ex: `width: 100%;` when flag is enabled